src/main/java/uk/gov/hmcts/reform/preapi/media/MediaKind.java

```diff
@@ -325,10 +326,10 @@ public List<LiveEventDTO> getLiveEvents() {
     @SuppressWarnings("checkstyle:VariableDeclarationUsageDistance")
     public RecordingStatus stopLiveEvent(CaptureSessionDTO captureSession, UUID recordingId)
         throws InterruptedException {
-        var recordingNoHyphen = getSanitisedId(recordingId);
+        var recordingNoHyphen = getSanitisedLiveEventId(recordingId);
         var recordingTempAssetName = recordingNoHyphen + "_temp";
         var recordingAssetName = recordingNoHyphen + "_output";
-        var captureSessionNoHyphen = getSanitisedId(captureSession.getId());
+        var captureSessionNoHyphen = getSanitisedLiveEventId(captureSession.getId());
```

Comment:
Could you update the `getSanitisedLiveEventId` method that is being imported from `MediaResourcesHelper` to be the same as the Azure one `.substring(0, 24)`

Reply:
ha, yeh just saw that too :) thanks